README.md

```diff
@@ -117,6 +118,34 @@ It's also possible to define your own linters to add to the built-in list. These
 ]
 ```
 
+
+## Code examples
+
+### Setting custom configuration path
+
+```js
+var path = require('path');
+var fs = require('fs');
+var LessHint = require('lesshint');
+var lesshint = new LessHint();
+
+var lesshintConfigPath = path.resolve('my-custom-path/.lesshintrc');
+let lesshintConfig = JSON.parse(fs.readFileSync(lesshintConfigurePath, 'utf8'));
+
+lesshint.configure(lesshintConfiguration);
+```
+
+### Using the default reporter
```

Comment:
Could we merge this with "Using reporters from code" further down?

Cause I noticed know that that example is wrong so we might as well replace it with this one and a comment explaining that the reporter name is optional.